.github/workflows/build.yml

```diff
@@ -20,18 +20,20 @@ jobs:
       fail-fast: false
       matrix:
         major_version: [37, 38]
-        driver_version: [520, 525, 530, 470]
+        driver_version: [470, 520, 525, 530, 535]
         exclude:
-          - driver_version: 525
-            major_version: 37
-          - driver_version: 525
+          - driver_version: 470
             major_version: 38
           - driver_version: 520
             major_version: 37
           - driver_version: 520
             major_version: 38
-          - driver_version: 470
+          - driver_version: 525
+            major_version: 37
+          - driver_version: 525
             major_version: 38
```

Comment:
My OCD needed to fix the ordering of these...  Both lists are now from oldest to newest :)